JenkinsMaster.groovy

```diff
@@ -0,0 +1,76 @@
+pipeline {
+    agent {
+        label 'docker01'
+    }
+    options {
+        ansiColor colorMapName: 'XTerm'
+    }
+    parameters {
+        string(
+            name: 'APP_SERVERS',
+            defaultValue: '10.10.10.122',
+            description: 'Deploy container to these servers. List of servers separated by comma.'
+        )
+    }
+    stages {
+        stage('Build') {
+            steps {
+                sh "docker build --rm=true -t captain-hook-master ."
+            }
+        }
+
+        stage('Prepare and upload to registry ') {
+            steps {
+                withCredentials([string(credentialsId: 'docker-registry-azure', variable: 'DRpass')]) {
+                    sh 'docker login roihunter.azurecr.io -u roihunter -p "$DRpass"'
+                    sh "docker tag captain-hook-master roihunter.azurecr.io/captain-hook/master"
+                    sh "docker push roihunter.azurecr.io/captain-hook/master"
+                    sh "docker rmi captain-hook-master"
+                    sh "docker rmi roihunter.azurecr.io/captain-hook/master"
+                }
+            }
+        }
```

Comment:
@Joozty It would be nice if building the image was optional :-) You can see how @jimrkh has it here: https://github.com/business-factory/lady-image/pull/547